saleor/dashboard/category/forms.py

```diff
@@ -4,24 +4,35 @@
 from django.utils.translation import pgettext_lazy
 from text_unidecode import unidecode
 
+from ...core.utils import merge_dicts
 from ...product.models import Category
+from ..seo.utils import (
+    MIN_DESCRIPTION_LENGTH, MIN_TITLE_LENGTH, SEO_HELP_TEXTS, SEO_LABELS,
+    SEO_WIDGETS)
 
 
 class CategoryForm(forms.ModelForm):
     def __init__(self, *args, **kwargs):
         self.parent_pk = kwargs.pop('parent_pk')
-        super(CategoryForm, self).__init__(*args, **kwargs)
+        super().__init__(*args, **kwargs)
+        self.fields['seo_description'].widget.attrs.update({
+            'data-bind': self['description'].auto_id,
```

Comment:
It's hard to guess the purpose of this attribute by reading this name. It would be great if we could come up with a bit more descriptive name.